app/controllers/items/partial_move.js

```diff
@@ -32,6 +35,19 @@ export default Ember.Controller.extend({
       }else{
         this.transitionToRoute('items.search_location', item.id, { queryParams: { isPartialMove: true }});
       }
+    },
+    editPackagesLocation(packagesLocation){
```

Comment:
In ECMA6, it is recommended that we use functionName(){..} and not functionName: function() {...}
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes